src/screens/SUE/SueReportScreen.tsx

```diff
@@ -288,6 +348,8 @@ export const SueReportScreen = ({
           const images = JSON.parse(getValues().images);
 
           let totalSize = 0;
+          const totalSizeLimit = parseInt(limitation?.maxAttachmentSize?.value) || 31457280;
```

Comment:
we have defaults in the code, right. so the hardcoded `31457280` should come from that defaults i guess.